### PR TITLE
web/api/performanceobserver h2m Performance observer

### PR DESCRIPTION
**Pull request by fwqaaq**
*  It still has fields that are not modem In browser compatibility when I use `yari` to convert html to md
* `{{Compat("api.PerformanceObserver.PerformanceObserver")}}`

### CODE REVIEW

**Comment by yin1999** [comment]
Hi @Jack-Zhang-1314, cloud you change the first commit (4ec0459) message?

File: files/zh-cn/web/api/performanceobserver/index.md [hunk 1]
```diff
@@ -8,50 +8,35 @@ tags:
   - 接口
 translation_of: Web/API/PerformanceObserver
 ---
-<div>{{APIRef("Performance Timeline API")}}</div>
+{{APIRef("Performance Timeline API")}}**`PerformanceObserver`** 用于监测性能度量事件，在浏览器的性能时间轴记录下一个新的 {{domxref("PerformanceEntry","performance entries", '', 'true')}} 的时候将会被通知 。{{AvailableInWorkers}}
```
Comment (yin1999) [inline files/zh-cn/web/api/performanceobserver/index.md:11]: please spread into three lines:

```suggestion
{{APIRef("Performance Timeline API")}}

**`PerformanceObserver`** 用于监测性能度量事件，在浏览器的性能时间轴记录下一个新的 {{domxref("PerformanceEntry","performance entries", '', 'true')}} 的时候将会被通知 。

{{AvailableInWorkers}}
```

**Comment by Jack-Zhang-1314** [comment]
> Hi @Jack-Zhang-1314, cloud you change the first commit ([4ec0459](https://github.com/mdn/translated-content/commit/4ec04598ade0e909a3a858a468c807607e388f57)) message?

yeah, to save the previous `commit` information

**Comment by yin1999** [comment]
> > Hi @Jack-Zhang-1314, cloud you change the first commit ([4ec0459](https://github.com/mdn/translated-content/commit/4ec04598ade0e909a3a858a468c807607e388f57)) message?
> 
> yeah, to save the previous `commit` information

message means `E:/github/git/zh-cn/web/api/performanceocserver convert html to md`, could you use relative path in message?

**Comment by yin1999** [comment]
> message means `E:/github/git/zh-cn/web/api/performanceocserver convert html to md`, could you use relative path in message?

And, please squash the last two commits

**Comment by Jack-Zhang-1314** [comment]
> > > Hi @Jack-Zhang-1314, cloud you change the first commit ([4ec0459](https://github.com/mdn/translated-content/commit/4ec04598ade0e909a3a858a468c807607e388f57)) message?
> > 
> > 
> > yeah, to save the previous `commit` information
> 
> message means `E:/github/git/zh-cn/web/api/performanceocserver convert html to md`, could you use relative path in message?

* could you modify the commit message?
* And using `git gc` to squash?

**Comment by yin1999** [comment]
> * could you modify the commit message?

yes

> * And using `git gc` to squash?

See: <https://docs.github.com/en/get-started/using-git/about-git-rebase#commands-available-while-rebasing>, `squash` command

**Comment by Jack-Zhang-1314** [comment]
like this?

**Comment by yin1999** [comment]
> like this?

Yes, and please modify the commit message

**Comment by Jack-Zhang-1314** [comment]
> Yes, and please modify the commit message

I don't know how to modify the first commit message? could you help me

**Comment by yin1999** [comment]
> I don't know how to modify the first commit message? could you help me

https://docs.github.com/cn/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/changing-a-commit-message

**Comment by Jack-Zhang-1314** [comment]
> > I don't know how to modify the first commit message? could you help me
> 
> https://docs.github.com/cn/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/changing-a-commit-message

It needs to checkout the commit. I don't expect its documentation to be so brief

**Pull request merged**